core/jazz_environment-event-handler/config/dev-config.json

```diff
@@ -54,5 +54,15 @@
     "website": "build-pack-website",
     "sls-app": "build-pack-sls-app",
     "ui": "jazz_ui"
+  },
+  "TVAULT": {
+    "IS_ENABLED": "{tvault-enabled}",
+    "API": "/jazz/t-vault/safes",
+    "ENDPOINT": "/t-vault/safes/",
+    "HOSTNAME": "{tvault-host-name}",
+    "USERS": "/jazz/usermanagement/users",
+    "ADD_ADMINS": "/user",
+    "ENV_DETAILS": "/prod/jazz/environments",
```

Comment:
Should this be `xxxxxxx-dev`?

Reply:
This should be just dev.

Reply:
@dmpl1 Shouldn't this just be `/dev/jazz/environments`?

Reply:
Addressed with /dev/jazz/environments